core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSourceOffset.scala

```diff
@@ -22,6 +22,7 @@ import com.fasterxml.jackson.databind.annotation.JsonDeserialize
 import org.json4s._
 import org.json4s.jackson.JsonMethods.parse
 
+import org.apache.spark.sql.connector.read.streaming
```

Comment:
can you rename read.streaming.Offset to OffsetV2 to make it more clear

Reply:
renamed in here, a similar thing is done in DeltaSource, wasn't sure if I should update all those too